src/ado/reprun.ado

```diff
@@ -255,22 +262,52 @@ end
         * Not part of a multiline line
         else {
 
-          *Reset the last line local
+          * Reset the last line locals
           local last_line = ""
           local line_command = "OTHER"
           local dofile ""
           local doflag 0
-          foreach w in `macval(line)' {
-            get_command, word("`w'")
+          local looptype ""
+          local loopflag 0
+
+          // Sanitize that string! -- see d17586d873a978987f34ba2fe536a311107ea58b for more regex
+          local theline = `"`macval(line)'"'
+          while regexm(`"`macval(theline)'"',"[\*]") {
+            local theline = regexr(`"`macval(theline)'"',"[\*]","")
+          }
+          while regexm(`"`macval(theline)'"',"\[//]"){
+            local theline = regexr(`"`macval(theline)'"',"\[//]","")
+          }
+
+          // Identify all commands in line
+          foreach w in `macval(theline)' {
+            cap get_command, word(`"`w'"')
```

Comment:
@kbjarkefur This is the best I could do here -- I cannot fix a single `"` yet.

Reply:
I'm still not 100% sure what an code example would be that is valid Stata code that have a single `"`. is there an example in the test files?

Reply:
This is one of the examples:
```
/* the do-file tests the "single quote
error"	
*/	
	sysuse auto, clear
		
	bys foreign: egen assigned = total(check)
	
	
```

Reply:
Similarly Ankriti had a file that included a line like:

```
* A very badly behaved comment "
```

Reply:
Here’s another example that produces the same error. However, it ran without issues in the previous version.
```
*** Load data (
	sysuse auto, clear
```

Reply:
Yes, that’s the `(` I suppose. Last time I wiped many more characters but I tried to minimize it this time. Can you provide a list of all characters which cause failure in this kind of example? You should be able to implement the fix since the code is templated right?